src/app/games/apagos/ApagosTutorial.ts

```diff
@@ -0,0 +1,69 @@
+import { ApagosMove } from 'src/app/games/apagos/ApagosMove';
+import { ApagosState } from 'src/app/games/apagos/ApagosState';
+import { TutorialStep } from 'src/app/components/wrapper-components/tutorial-game-wrapper/TutorialStep';
+import { ApagosCoord } from './ApagosCoord';
+import { Player } from 'src/app/jscaip/Player';
+import { MGPValidation } from 'src/app/utils/MGPValidation';
+
+export class ApagosTutorial {
+
+    public tutorial: TutorialStep[] = [
+        TutorialStep.informational(
+            $localize`Initial board and goal of the game`,
+            $localize`At Apagos, there are 4 squares, each of them have a fix number of holes to put pieces. Each player start with 10 pieces. Dark pieces belong to first player, light pieces belong to the second one. The game ends when no one can play. The player owning the most pieces in the righter square wins!`,
```

Comment:
Je vois plus mon commentaire précédent, mais:
fixed*
starts*
belong to the* first player
rightmost*

Reply:
fait!